manifests/linux/redhat.pp

```diff
@@ -140,6 +140,11 @@
         case $::operatingsystem {
           'CentOS': {
             case $::operatingsystemrelease {
+              /^5\..*/: {
```

Comment:
Why is this part of this PR? Has nothing to do with SLES 11.

Reply:
This was removed by accident in an earlier pr

Reply:
Still has nothing to do with this PR. Either revert the other PR that introduced it or separate it into its own commit. This doesn't make for good git history and would be hard to track.

Reply:
Will separate it out into a 2nd commit